content/es/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,748 @@
+ninguna de las reglas las decidirá el controlador ingress (consulta la
+documentación de tu controlador de ingress para saber cómo maneja este caso).
+
+Si ninguno de los hosts o rutas coincide con la petición HTTP en los objetos Ingress,
+el tráfico será enrutado a tu backend predeterminado.
+
+
+Un recurso `Resource` backend es una referencia de objeto
+(ObjectRef en inglés)
+a otro recurso de Kubernetes dentro del mismo espacio de nombres que el objeto
+Ingress.
+Este `Resource` es mutuamente exclusivo con el Service,
+y la validación fallará si ambos se especifican.
+Un uso común para un `Resource` backend es para ingresar datos a un backend de almacenamiento de datos con activos estáticos.
+
+{{% code_sample file="service/networking/ingress-resource-backend.yaml" %}}
+
+Luego de crear el Ingress mencionado arriba,
+puedes verlo con el siguiente comando:
+
+
+```bash
+kubectl describe ingress ingress-resource-backend
+```
+
+```
+Name:             ingress-resource-backend
+Namespace:        default
+Address:
+Default backend:  APIGroup: k8s.example.com, Kind: StorageBucket, Name: static-assets
+Rules:
+  Host        Path  Backends
+  ----        ----  --------
+  *
+              /icons   APIGroup: k8s.example.com, Kind: StorageBucket, Name: icon-assets
+Annotations:  <none>
+Events:       <none>
+```
+
+### Tipos de ruta
+
+Se requiere que cada ruta de un Ingress tenga un tipo de ruta correspondiente.
+Las Rutas que no incluyen un `pathType` explícito fallarán la validación.
+Hay 3 tipos de rutas soportadas:
+
+* `ImplementationSpecific`: Con este tipo de ruta, la coincidencia depende de la
+  IngressClass.
+  Las implementaciones pueden tratar esto como un `pathType` separado o
+  tratarlas de forma idéntica a los tipos de ruta `Prefix` o `Exact`.
+
+* `Exact`: Coincide la ruta de la URL exactamente con sensibilidad a mayúsculas
+  y minúsculas.
+
+* `Prefix`: Coincide basado en el prefijo de una ruta URL dividida por `/`.
+  La coincidencia es sensible a mayúsculas y minúsculas y hecha en un elemento
+  de la ruta por elemento.
+  Un elemento de la ruta refiere a la lista de etiquetas in la ruta dividida por
+  el separador `/`.
+  Una petición es una coincidencia para la ruta _p_ si cada _p_ es un elemento
+  prefijo de _p_ de la ruta requerida.
+
+{{< note >}}
+Si el último elemento de una ruta es una cadena de caracteres del último
+elemento de la ruta elemento, no es una coincidencia
+(por ejemplo: `/foo/bar`
+coincide con `/foo/bar/baz`, pero no coincide con `/foo/barbaz`).
+{{< /note >}}
+
+### Ejemplos
+
+| Tipo     | Ruta(s)                           | Ruta de la(s) peticion(es) | ¿Coincide?                             |
+|----------|-----------------------------------|----------------------------|----------------------------------------|
+| Prefijo  | `/`                               | (todas las rutas)          | Si                                     |
```

Comment:
Add an _"acento"_ to each `Sí`, since this is a response to the question of _"¿Coincide?"_ and these affirmative answers always need to have _"acento"_ (for reference, see the [RAE dictionary of "sí"](https://dle.rae.es/s%C3%AD))

```suggestion
| Prefijo  | `/`                               | (todas las rutas)          | Sí                                     |
```